IDR/production/metadata_extraction/extraction_utils/clean_channels.py

```diff
@@ -0,0 +1,53 @@
+import re
+
+
+def clear_parentheses(channel_item):
+    if bool(channel_item[channel_item.find("(") + 1 : channel_item.rfind(")")]):
+        channel_item = re.sub(r"\([^()]*\)", "", channel_item)
+
+    return channel_item.strip()
+
+
+def clean_channel(channels):
```

Comment:
Maybe I am missing something, but what about different capitalization? `DAPI` should be considered differently from `dapi`, for instance

Reply:
I didn't even think about that. I standardized all channel values to lower case.

Added in a0fff2e5a91230caf2684862aeaef911ececa152.